extension/changelog.json

```diff
@@ -1,4 +1,15 @@
 [
+	{
+		"version": { "major": 7, "minor": 5, "build": 3 },
+		"title": "Beta",
+		"date": false,
+		"logs": {
+			"features": [],
+			"fixes": [],
+			"changes": [{ "message": "Option to highlight items in Bazaars that have a ccost less than the vendor price.", "contributor": "TravisTheTechie" }],
+			"removed": []
+		}
+	},

```

Comment:
Just add an entry in the existing version, first pull main though as otherwise you'll conflict.